native_client/BUILD

```diff
@@ -113,7 +122,13 @@ cc_library(
         "kenlm/lm/*.hh",
         "kenlm/util/*.hh",
     ]),
-    deps = [":kenlm_import"],
+    deps = select({
+        # This needs to be linked statically when using WASM/Emscripten.
+        # This has licensing implications since KenLM is LGPL.
```

Comment:
Licensing implications means we have to make licensing changes, in this case we need to include the LGPL text inside the WASM bindings package.